<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stack">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="courses">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Выберите группу:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="addgroupButton">
          <property name="text">
           <string>Добавить группу</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="removeGroupsButton">
          <property name="text">
           <string>Удалить группы</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QListView" name="course1List"/>
          </item>
          <item row="1" column="1">
           <widget class="QListView" name="course2List"/>
          </item>
          <item row="1" column="4">
           <widget class="QListView" name="course5List"/>
          </item>
          <item row="1" column="2">
           <widget class="QListView" name="course3List"/>
          </item>
          <item row="1" column="3">
           <widget class="QListView" name="course4List"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>1 курс</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>2 курс</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>3 курс</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>4 курс</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>5 курс</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="subjects">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="SubjectListEdit" name="subjectListEdit" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>437</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SubjectListEdit</class>
   <extends>QWidget</extends>
   <header>SubjectListEdit.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>addgroupButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAddGroupDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeGroupsButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeGroups()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showAddGroupDialog()</slot>
  <slot>removeGroups()</slot>
 </slots>
</ui>
